distribution/docs/src/main/jdocs/content/_tables/org.codice.ddf.catalog.security.ResourceUriPolicy-table-contents.adoc

```diff
@@ -0,0 +1,29 @@
+:type: documentation
+:status: published
+
+.[[org.codice.ddf.catalog.security.ResourceUriPolicy]]Resource URI Policy Plugin
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Permit Resource URI on Creation
+|createPermissions
+|String
+|Allow users to provide a resource URI when creating a metacard
```

Comment:
❓ Should these descriptions say _users with the specified claim_? Can there be more than one claim?